src/documentmanager/documentmanager.cpp

```diff
@@ -325,43 +333,39 @@ QMdiArea* DocumentManager::lastAreaInFocus()
     // search for area which accomodates last doc in focus
     for (const auto& area: mDocAreas)
```

Comment:
`for (const auto& area: mDocAreas)`
rewrite it to
`for (const auto &area: mDocAreas)`

Reply:
fixed